cmd/forklift-controller/main.go

```diff
@@ -19,6 +19,8 @@ package main
 import (
 	"net/http"
 	"os"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
```

Comment:
the location of these imports looks wrong - did the IDE placed them there?

Reply:
i think so

Reply:
ok, that's a bit not standard and doesn't align with the rest of our code - standard imports are grouped together before the other imports. I've changed it but please take a look at the settings of the IDE then to see how to change it